addons/web/static/src/core/currency.js

```diff
@@ -18,7 +18,15 @@ export async function getCurrencyRates() {
     const rates = reactive({});
 
     function recordsToRates(records) {
-        return Object.fromEntries(records.map((r) => [r.id, r.inverse_rate]));
+        return Object.fromEntries(
+            records.map((r) => [
+                r.id,
+                {
+                    rate: r.inverse_rate,
+                    date: toLocaleDateString(parseDate(r.date)),
```

Comment:
we typically manipulate luxon instances in the webclient, and call the method to format them when we need. So here I would suggest to store `date: parseDate(r.date)`, and in the multi currency popover, to call `toLocaleDateString`